src/ALZ/Assets/alz-bicep-config/v0.14.1-pre.config.json

```diff
@@ -332,6 +340,7 @@
         },
         "AllSubscriptionIds": {
             "Type": "Computed",
+            "Process": "@($args | Select-Object -Unique)",
```

Comment:
Might want to consider casing here...  Perhaps tolower on the inputs before we test for uniqueness...